### PR TITLE
Fix documents created in DEV and watermark applied to documents

### LINKED ISSUE #151: In case of target env "D" & "assembly mode" - create ONLY necessary docs, rather than all available ones
In case of environment = `D` and version `WIP` we should create all docs, so people can really check them out. 

However in case of `project.assembleMode` - we should only create the really relevant docs for  `D` and `Q`  .. and not all docs possible. 

This way we make it easy for the user to know what needs to go to EDMS vs. what's clutter...

### PR DESCRIPTION
**Pull request by georgfedermann**
closes #151

### CODE REVIEW

**Review by clemensutschig — DISMISSED** [review]

**Comment by metmajer** [comment]
@georgfedermann is your work still in progress? tested? Cheers!

**Comment by georgfedermann** [comment]
> @georgfedermann is your work still in progress? tested? Cheers!

only thing missing is running the changes in a pipeline, which is happening right now ...

**Review by metmajer — DISMISSED** [review]

**Comment by metmajer** [comment]
Thanks @georgfedermann, please squash and merge to master.

File: src/org/ods/scheduler/LeVADocumentScheduler.groovy [hunk 1]
```diff
@@ -10,70 +10,70 @@ class LeVADocumentScheduler extends DocGenScheduler {
     // Document types per GAMP category
     private static Map GAMP_CATEGORIES = [
         "1": [
+            LeVADocumentUseCase.DocumentType.CFTP as String,
+            LeVADocumentUseCase.DocumentType.CFTR as String,
             LeVADocumentUseCase.DocumentType.CSD as String,
-            LeVADocumentUseCase.DocumentType.RA as String,
```
Comment (clemensutschig) [inline src/org/ods/scheduler/LeVADocumentScheduler.groovy]: I strongly vouch to leave the ordering as is .. (based on v-model) ...

Comment (georgfedermann) [reply src/org/ods/scheduler/LeVADocumentScheduler.groovy]: ok, will revert the order. was just easier to search through them that way/

**Review by clemensutschig — APPROVED** [review]
top!

**Pull request merged**